### PR TITLE
Internal: Stabilize widgets & accordion tests + some alignment

### PR DESCRIPTION
**Pull request by yotamselementor**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
     needs: [build-plugin]
     strategy:
       matrix:
-        testSuite: ['default', 'nested-tabs', 'reverse-columns']
+        testSuite: ['elements-regression', 'default', 'nested-tabs', 'reverse-columns']
```
Comment (TzviRabinovitch) [inline .github/workflows/playwright.yml:52]: if possible lets keep the default the 1st item

File: tests/playwright/sanity/assets/dev/js/editor/elements/views/widget.test.js [hunk 2]
```diff
@@ -9,7 +9,9 @@ test.describe( `$e.run( 'editor/elements/views/widget' )`, () => {
 
 		// Act.
 		await editor.addWidget( 'image-carousel' );
-		await editor.getPreviewFrame().waitForSelector( '.elementor-widget .elementor-widget-empty-icon' );
+		await editor.getPreviewFrame().waitForSelector( '.elementor-widget-image-carousel .elementor-widget-empty-icon', {
+			state: 'visible',
```
Comment (TzviRabinovitch) [inline tests/playwright/sanity/assets/dev/js/editor/elements/views/widget.test.js:13]: state: 'visible',

not required, this is the default

**Review by TzviRabinovitch — CHANGES_REQUESTED** [review]

File: .github/workflows/playwright.yml (hunk 1 above)
Comment (yotamselementor) [reply .github/workflows/playwright.yml:52]: I did this on purpose - `elements-regression` is the only one here not in the "usual" framework, so I wanted it to stand out.
If I had placed it last, the next test would come after it.
Another option, would be: `['default', 'elements-regression', 'nested-tabs', 'reverse-columns']`

What do you think?

File: tests/playwright/sanity/assets/dev/js/editor/elements/views/widget.test.js (hunk 2 above)
Comment (yotamselementor) [reply tests/playwright/sanity/assets/dev/js/editor/elements/views/widget.test.js]: https://github.com/elementor/elementor/pull/21559/commits/516fb74dfbd83b4de876f1c3bb1f4f8c63b72949

**Review by TzviRabinovitch — APPROVED** [review]

**Pull request merged**